pkg/subctl/cmd/join.go

```diff
@@ -106,8 +121,13 @@ func addJoinFlags(cmd *cobra.Command) {
 		"List of domains to use for multicluster service discovery")
 	cmd.Flags().StringSliceVar(&imageOverrideArr, "image-override", nil,
 		"Override component image")
+	cmd.Flags().BoolVar(&healthCheckEnable, "health-check", true,
+		"enable Gateway health check")
```

Comment:
This one hasn't released yet so I think we should just change this for 0.8.0 rather than deprecating.

Reply:
Good point, I’ll split it out so we can merge it for 0.8.

Reply:
https://github.com/submariner-io/submariner-operator/pull/975